.github/nfpm-configs/README.md

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I don't love this living under .github, but we don't currently have a build directory, or something similar. If we accumulate more things like, we should consolidate them in one top level place along with the docker stuff.

Reply:
I agree it makes more sense to reserve `.github` for GitHub. :slightly_smiling_face: I was thinking "only GH will run this," but it's also possible someone wants to run it locally to test a package change:

```bash
ZITI_VERSION=0.30.0 \
ZITI_HOMEPAGE="https://openziti.io" \
ZITI_VENDOR=netfoundry \
ZITI_MAINTAINER="Maintainers <developers@openziti.org>" \
GOARCH=amd64 \
    nfpm pkg --packager deb --target  . --config linux-packages/nfpm-linux-ziti-cli.yaml
```

I've moved those files out of ".github" dir to a new top-level dir "linux-packages."

Reply:
Should the top level be something like dist, or packages with a subfolder for linux?

Reply:
"packages" might be confusing because Go has "packages" too, and "dist" doesn't have a clear meaning by itself. How about "build/dist-packages/linux?" Then I can send another pull request to nest "build/docker-images." That'll keep further install/distribution packages organized like "build/dist-packages/homebrew."

Reply:
Sounds good to me. Nice work on all this @qrkourier